test/precompile.jl

```diff
@@ -42,8 +42,10 @@ precompile_test_harness("Inference caching") do load_path
 
         @setup_workload begin
             @compile_workload begin
-                autodiff(Reverse, mul, Active, Active(1.0), Active(2.0))
-                autodiff(Forward, mul, Duplicated, Duplicated(1.0, 1.0), Const(2.0))
+                autodiff(ReverseMode{false,InlineABI,false}(), mul, Active, Active(1.0), Active(2.0))
+                # Non-Inline mode uses `@generated` functions and poisons the caller
```

Comment:
inline mode also uses generated functions, but it uses an llvmcall not a call to custom jit